src/util/is-docker.ts

```diff
@@ -0,0 +1,30 @@
+// is ESM
+// https://github.com/sindresorhus/is-docker/blob/main/index.js
```

Comment:
this dep is ESM so I go compilation errors.
Can probably fixed with the right webpack config but copy pasting the code was just as fast :sweat_smile: